### ISSUE TITLE
Overview page tooltips should be interactive

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#10263]
### Description:

Overview page tooltips should be in interactive type
![image](https://user-images.githubusercontent.com/20179540/111123355-3cb44300-8595-11eb-899c-a9995acf0cbf.png)

`<Tooltip interactive>`

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
